xapian-core/matcher/extraweightpostlist.cc

```diff
@@ -33,11 +33,12 @@ ExtraWeightPostList::get_weight(Xapian::termcount doclen,
 				Xapian::termcount unique_terms,
 				Xapian ::termcount wdfdocmax) const
 {
-    /* Weight::get_sumextra() takes two parameters (document length and number
-     * of unique terms) but none of the currently implemented weighting schemes
-     * actually use the latter - it was added because it's likely to be wanted
-     * at some point, and so that we got all the incompatible changes needed to
-     * add support for the number of unique terms over with in one go.
+    /* Weight::get_sumextra() takes three parameters (document length, number of
+     * unique terms and the max wdf in the document) but none of the currently
+     * implemented weighting schemes actually use the max wdf - it was added
+     * because it's likely to be wanted at some point, and so that we got all
+     * the incompatible changes needed to add support for the max wdf over with
+     * in one go.
```

Comment:
What it said before about unique terms being unused is still correct - of the parameters, only `doclen` is currently used (which is perhaps the simpler way to put it now).

Reply:
I have corrected it. (I was confused as we have TfIdf normalisations that use unique terms.)